arrays_strings/str_diff/str_diff_solution.ipynb

```diff
@@ -76,7 +76,7 @@
     "\n",
     "Complexity:\n",
     "* Time: O(m+n), where m and n are the lengths of s, t\n",
-    "* Space: O(1), for the dict, where h is the unique chars in s"
+    "* Space: O(1), for the result, where the result takes always 1 space"
```

Comment:
Nice catch!

For this line, I think "result takes always 1 space" might be worded a little differently.

Or...I wonder if we can make it more concise (without loss of detail) with just the following:

>* Space: O(1)